.github/actions/stale_issues/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Filter stale issues
+inputs:
+  token:
+    description: 'The GITHUB_TOKEN secret'
+    required: true
+  period:
+    description: "The stale period in days"
+    required: false
+    type: number
+    default: 14
+  repo:
+    description: "The target repository"
+    required: false
+    type: string
+    default: "${{ github.repository_owner }}/${{ github.event.repository.name }}"
+outputs:
+  result:
+    description: "The result of the query"
+    value: ${{ steps.found_issues.outputs.result }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get period date
+      id: date
+      shell: bash
+      run: |
+        period_date=$(date --date "${{ inputs.period }} days ago" --utc +"%Y-%m-%dT%H:%M:%SZ")
+        echo "period=$period_date" >> $GITHUB_OUTPUT
+    - name: Get inactive open issues
+      id: stale_issues
+      if: ${{ success() }}
```

Comment:
Isn't this implied? I thought that a failure always interrupts the workflow so you do not have to do that.

Reply:
Indeed. I will remove.